### ISSUE TITLE
Replace validate_email with email-validator

### ISSUE THREAD

**Issue by bgermann** [crossbario/crossbar#2030]
validate_email seems to be abandoned. Please consider replacing it with the alternative module email-validator.
This will help me packaging the latest crossbar version in Debian because validate_email does not have a chance to enter Debian.

**Comment by oberstet** [comment]
ok, just to be sure, we're talking about

https://github.com/crossbario/crossbar/blob/71787f9eae220966e6be4a608cc049e336f9a85b/requirements-min.txt#L73

and thus replace 

https://pypi.org/project/validate_email/
https://github.com/syrusakbary/validate_email

with the following

https://pypi.org/project/email-validator/
https://github.com/JoshData/python-email-validator

right?

**Comment by bgermann** [comment]
Right. I guess they do not have the same API, so there would be additional changes.
